crates/node/src/catchup.rs

```diff
@@ -213,6 +227,9 @@ impl Node {
                     batch_size: self.network_client.catchup_config.batch_size,
                     url: None,
                     hash: None,
+                    // In this situation, where we have no context, we have no way to know the
+                    // public key.
+                    public_key: PublicKey([0; 32]),
```

Comment:
`ContextManager::join_context` should take an identity for the member who wants to join, so at this point we should have one.